pyiron_atomistics/lammps/base.py

```diff
@@ -1398,6 +1085,28 @@ def _set_selective_dynamics(self):
                             "velocity___constraintz"
                         ] = "set NULL NULL 0.0"
 
+    def _get_rotation_matrix(self, pressure):
```

Comment:
`_get_rotation_matrix()` is moved up to have all the static functions at the bottom of the class.